### ISSUE TITLE
imagenet训练代码可以公布吗？

### ISSUE THREAD

**Issue by brisker** [huawei-noah/AdderNet#50]
imagenet训练代码可以公布吗？

**Comment by HantingChen** [comment]
You can directly modify the Pytorch official imagenet training example: https://github.com/pytorch/examples/blob/master/imagenet/main.py, by replacing the conv-resnet with adder-resnet and using cosine learning rate decay with 150 epochs training.
